src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -49,6 +129,17 @@ public enum JsonTokenType : byte
         String = (byte)6,
         True = (byte)8,
     }
+    public enum JsonValueType : byte
+    {
+        Array = (byte)2,
+        False = (byte)6,
+        Null = (byte)7,
```

Comment:
is there any benefit to having Null be (byte)0

Reply:
No, Null represents the literal token "null".  "Undefined" is the Javascript "implicit" value, and thus the best equivalent for "there's no value here" in this enum (which is why it got zero).